### DISCUSSION TITLE
How can I show the changelog in Verdaccio?

### DISCUSSION THREAD

**Discussion by nicothin** [verdaccio/verdaccio#4043] [Q&A] [answered]
The Verdaccio web interface shows (on the package page) README.md, but does not show changelog.md. Can I somehow see the contents of changelog.md there?

**Comment by mbtools** [comment, marked as answer]
The changelog is not part of the npm metadata. However, you can just follow the link to the repository of a package and find the changelog there:

![image](https://github.com/verdaccio/verdaccio/assets/59966492/0deb9d80-8cca-4883-9170-7d4eadc8fc17)
